### DISCUSSION TITLE
How to get access to drain event when built up Backpressure is gone

### DISCUSSION THREAD

**Discussion by OmGuptaIND** [uNetworking/uWebSockets.js#1009] [Q&A]
When backpressure is there, I cannot, in any case, drop the peer from the service. That will be a serious issue. I want to save the dropped messages and then resent them to the peer when the backpressure is drained. 

Can I have some event emitted for such a use case? Also if you can help me with should I even do this

In my case, the backpressure is set to 1MB. For my case I do have some serious payload size

**Comment by lagden** [comment]
Here:
https://unetworking.github.io/uWebSockets.js/generated/interfaces/WebSocketBehavior.html#dropped
